infra/modules/providers/azure/app-service/main.tf

```diff
@@ -103,9 +103,10 @@ resource "azurerm_app_service_slot" "appsvc_staging_slot" {
 }
 
 data "azurerm_app_service" "all" {
-  count               = length(azurerm_app_service.appsvc)
+  count               = length(azurerm_app_service_slot.appsvc_staging_slot)
```

Comment:
how do we know how many app services there are now?

Reply:
Glad you saw this! You're right, this data block drives the last and final output of the iso template. I'll revert this back and look for any of the consequences to that.

Reply:
@TechnicallyWilliams any update on this?

Reply:
Yeah, i'm making changes locally at the moment. May need to rebase with master as well.